python/nwis_client/README.md

```diff
@@ -54,3 +54,93 @@ print(observations_data.head())
 3 2019-08-01 04:45:00    streamflow       01646500            ft3/s  4170.0        [A]       0
 4 2019-08-01 05:00:00    streamflow       01646500            ft3/s  4170.0        [A]       0
 ```
+
+### Command Line Interface (CLI)
+The `hydrotools.nwis_client` package also includes a command-line utility that is installed to the environment targeted by the `pip` installation above. Starting a terminal and activating the correct Python environment should enable the `nwis-client` CLI. This tool will retrieve data from the USGS IV Web Service and write it to CSV format.
+
+This example demonstrates calling the help page:
+```bash
+$ nwis-client --help
+```
+```console
```

Comment:
1. New default is to output to stdout.
2. Added switches for comments and header.
3. Adjusted argument so it will switch to `click.get_text_stream` to read input from stdin. Successfully tested with direct terminal input, cat file, and pipe.